pysph/base/c_kernels.pyx.mako

```diff
@@ -12,6 +12,7 @@ CLASSES = (
     WendlandQuinticC4, WendlandQuinticC6, WendlandQuinticC2_1D,
     WendlandQuinticC4_1D, WendlandQuinticC6_1D
 )
+get_config().use_openmp = True;
```

Comment:
Why is this being done? Also, if you are changing it, should you not reset it to the original state? So my suggestion is to use the with_config or get the original value and restore it.